### PR TITLE
Add sorting capabilities to the command 'yarn extract'

### PR DESCRIPTION
**Pull request by carlosthe19916**
This PR allows `yarn extract` to sort alphabetically all key values from the internationalization .json files.

Currently the command `yarn extract` is used to generate all key/values in the internationalization .json files. The current configuration doesn't sort the key/values in the .json files generating some conflicts when a new PR with new key/values comes.

This doesn't change any behavior or functionality of the UI but just adds a better way of organizing the content of the internationalization files.

https://user-images.githubusercontent.com/2582866/112620535-f67aa180-8e28-11eb-8179-dd37e8299ca2.mp4

https://user-images.githubusercontent.com/2582866/112623853-275cd580-8e2d-11eb-91e8-44fe4bb8f777.mp4

### CODE REVIEW

File: src/pages/application-inventory/application-inventory.tsx [hunk 1]
```diff
@@ -410,9 +410,11 @@ export const ApplicationInventory: React.FC = () => {
             }
             noDataState={
               <NoDataEmptyState
+                // t('terms.applications')
```
Comment (mrizzi) [inline src/pages/application-inventory/application-inventory.tsx:413]: worth removing the comment?

File: src/pages/application-inventory/application-inventory.tsx [hunk 2]
```diff
@@ -410,9 +410,11 @@ export const ApplicationInventory: React.FC = () => {
...
                 title={t("composed.noDataStateTitle", {
                   what: t("terms.applications").toLowerCase(),
                 })}
+                // t('terms.application')
```
Comment (mrizzi) [inline src/pages/application-inventory/application-inventory.tsx:417]: worth removing the comment?

**Review by mrizzi — COMMENTED** [review]
Commented about removing some added comments that seems some tests done during development

File: src/pages/application-inventory/application-inventory.tsx (hunk 1 above)
Comment (carlosthe19916) [reply src/pages/application-inventory/application-inventory.tsx:413]: @mrizzi i18next-parser is the tool used for detecting which are the new "internationalization" keys and add them automatically to the .json files.

In this specific case the line 415 `what: t("terms.applications").toLowerCase(),` is not inside a React component neither is referenced directly in a JS file but it is part of a JSON parameter passed in line 414; as a result of this, the command `yarn extract` won't be able to detect `terms.applications`, adding a comment helps the tool to detect the key and add it to the .json internationalization files. For more info about this see https://github.com/i18next/i18next-parser#caveats

This doesn't mean that we need to add comments everywhere; whenever the user creates a commit the `husky` config will edit the files just like the video I shared in this PR and let the user know which are keys detected in your PR. So these 2 comments in this PR are needed to persist the key `terms.application`

Comment (mrizzi) [reply src/pages/application-inventory/application-inventory.tsx:413]: awesome, thanks @carlosthe19916 :+1:

File: src/pages/application-inventory/application-inventory.tsx (hunk 2 above)
Comment (mrizzi) [reply src/pages/application-inventory/application-inventory.tsx:417]: see above

**Review by mrizzi — APPROVED** [review]
Thanks @carlosthe19916 :+1:

**Pull request merged**